content/en/getting_started/integrations/_index.md

```diff
@@ -97,6 +97,15 @@ For example, setting `service` in your config file is the recommended [Agent set
 
 To better unify your environment, it is also recommended to configure the `env` tag in the Agent. To learn more, see [Unified Service Tagging][27].
 
+By default, the metrics reported by integrations include tags autodiscovered from the environment. For example, the metrics reported by a Redis check that runs inside a container include tags that refer to the container like `image_name`. This can be disabled by setting the `ignore_autodiscovery_tags` parameter to `false`:
```

Comment:
```suggestion
By default, the metrics reported by integrations include tags autodiscovered from the environment. For example, the metrics reported by a Redis check that runs inside a container include tags that refer to the container, such as `image_name`. You can turn this behavior off by setting the `ignore_autodiscovery_tags` parameter to `false`:
```

Reply:
Sorry, as you mentioned, the setting should be set to `true` instead of `false`.
I fixed that and also included your suggestion.